region/norfair/east.json

```diff
@@ -3348,14 +3380,24 @@
                   "notable": false,
                   "requires": [
                     "h_canNavigateHeatRooms",
-                      {"or": [
-                        "canWalljump",
-                        "HiJump",
-                        "SpaceJump",
-                        "canUseFrozenEnemies"
-                      ]},
+                    {"or": [
+                      "canWalljump",
+                      "HiJump",
+                      "SpaceJump",
+                      "canTrickyUseFrozenEnemies",
```

Comment:
Note this was "canUseFrozenEnemies" before, and we're changing it to "canTrickyUseFrozenEnemies" here. Also added a Spring Ball jump strat.